tools/fuchsia/devshell/serve.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+
+### Serves the out directory
+## usage:
+##   -p port for "pm serve" to listen on
+##   -d device_name to use for connecting
+##   -r If you are running a remote workflow
+##   --only-serve-runners If set will only serve the flutter and dart runners
+##   --out the out directory (ex: fuchsia_debug_x64)
```

Comment:
qq... if this is not currently used how does the package serving script disambiguate which runner to serve (ie. the one in out/fuchsia_debug_x64, out/fuchsia_profile_x64, or out/fuchsia_release_x64)?

Reply:
The all_packages.list file is a hardcoded list of all the possible packages for all types of builds. The incremental serving tool will watch for those files to show up so even if you switch build dirs it will pick up the change. This is not a good long term solution because it requires us to add new packages to this list whenever we add more. In a subsequent change I would like to have gn generate this list of packages which would require us specifying the out directory.

Reply:
ah that makes sense. thanks for clarifying!